tm2/pkg/iavl/nodedb.go

```diff
@@ -117,14 +117,6 @@ func (ndb *nodeDB) SaveNode(node *Node) {
 // Has checks if a hash exists in the database.
 func (ndb *nodeDB) Has(hash []byte) bool {
 	key := ndb.nodeKey(hash)
-
-	if ldb, ok := ndb.db.(*dbm.GoLevelDB); ok {
```

Comment:
Hey @jaekwon, something seems off here. `GoLevelDB.Has()` is essentially checking if `GoLevelDB.Get()` is not `nil`, making this condition alter the function's behavior to panic instead of returning false, specifically when using a particular driver.

Would you be alright with removing this code, or do you have a different approach in mind?

Reply:
PR should be ready after resolving this.